File: platforms/python/examples/scene/scene/set_default_light.py
from fragmentcolor import Light, Material, Mesh, Model, Scene, Vertex

mesh = Mesh()
mesh.add_vertex(
    Vertex.pbr([0.0, 0.5, 0.0]).set(Vertex.UV0, [0.5, 1.0]),
)
scene = Scene()
scene.add(Model(mesh, Material.pbr()))

key = Light.directional([0.3, -1.0, -0.4], [1.0, 0.95, 0.9])
scene.set_default_light(key)

File: platforms/kotlin/examples/scene/scene/set_default_light.kt
import org.fragmentcolor.*

val mesh = Mesh()
mesh.addVertex( Vertex.pbr(listOf(0.0f, 0.5f, 0.0f)).set("uv0", floatArrayOf(0.5f, 1.0f)), )
val scene = Scene()
scene.add(Model(mesh, Material.pbr()))

val key = Light.directional(listOf(0.3f, -1.0f, -0.4f), listOf(1.0f, 0.95f, 0.9f))
scene.setDefaultLight(key)